end_to_end_tests/metadata_snapshots/uv.pyproject.toml

```diff
@@ -3,7 +3,7 @@ name = "test-3-1-features-client"
 version = "0.1.0"
 description = "A client library for accessing Test 3.1 Features"
 authors = []
-requires-python = "~=3.9"
+requires-python = ">=3.9,<4.0"
```

Comment:
> When evaluating requires-python ranges for dependencies, uv only considers lower bounds and ignores upper bounds entirely. For example, `>=3.8, <4` is treated as `>=3.8`

I think we should consider dropping the upper bound from the template due to the various issues it's caused and that some tools ignore it anyway.

Reply:
Fine by me, seems like there will never be a Python 4 anyway 😋